ruby/examples/lib/register_user_with_form.rb

```diff
@@ -0,0 +1,39 @@
+require 'active_model'
+
+class RegisterUserWithForm
+  Request = Struct.new(:email, :name, :password)
+  Response = Struct.new(:form)
+
+  class Response::Form
```

Comment:
If this class was not here, feel like the dependencies are pointing the correct way.

Not sure what ActiveModel::Model does? Presume some magic for form_for?